tutorials/backup-postgresql-pgbackrest-s3/index.mdx

```diff
@@ -0,0 +1,256 @@
+- Differential backup: It only retrieves files that have changed since the last full backup. Both the full backup and the differential backup must be intact.
+- Incremental backup: It retrieves files that have changed since the last backup. The last backup can be a full backup or a differential backup.
+
+<Message type="requirement">
+
+- You have an account and are logged into the Scaleway console.
+- You have created instances running a PostgreSQL database on Ubuntu/Debian or RHEL/Rocky Linux. -> https://www.scaleway.com/en/docs/tutorials/back-up-postgresql-barman/
+- You have sudo privileges or access to the root user.
+- You have created an Object Storage bucket -> https://www.scaleway.com/en/docs/storage/object/how-to/upload-files-into-a-bucket
+
+</Message>
+
+
+## Installing pgBackRest
+
+The **PostgreSQL Global Development Group (PGDG)** provides an apt and yum repository. After importing the repository, you can install the PgBouncer package.
+
+For Ubuntu/Debian:
+1. Create the file repository configuration:
+    ```
+    sh -c 'echo "deb http://apt.postgresql.org/pub/repos/apt $(lsb_release -cs)-pgdg main" > /etc/apt/sources.list.d/pgdg.list'
+    ```
+
+2. Import the repository signing key:
+    ```
+    wget --quiet -O - https://www.postgresql.org/media/keys/ACCC4CF8.asc | apt-key add -
+    ```
+
+3. Update the package lists:
+    ```
+    apt update
+    ```
+4. Install the `pgbackrest` package
+    ```
+    apt install -y pgbackrest
+    ```
+
+
+For RHEL/Rocky Linux:
+1. Install the repository RPM:
+    ```
+    dnf install -y https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
+    ```
+
+2. Disable the built-in PostgreSQL module:
+    ```
+    dnf -qy module disable postgresql
+    ```
+
+3. Install the `pgbackrest` package
+    ```
+    dnf install -y pgbackrest
+    ```
+
+
+## Setting up pgBackRest
+
+1. Edit the `/etc/pgbackrest.conf` with the following changes (repository configurations for example purposes only):
+    ```
+    [global]
+    repo1-type=s3
+    repo1-s3-endpoint=<BUCKET-ENDPOINT>
+    repo1-s3-bucket=<BUCKET-NAME>
+    repo1-s3-key=<ACCESS-KEY>
+    repo1-s3-key-secret=<SECRET-KEY>
+    repo1-s3-region=<REGION>
+    repo1-path=/pgbackrest
+
+    repo1-retention-full=1
+    start-fast=y
+    archive-async=y
+
+    [db-primary]
+    pg1-path=/var/lib/pgsql/14/data
+    ```
+
+<Message type="note">
+
+PostgreSQL data directory is '/var/lib/postgresql/14/main' in Ubuntu/Debian by default
+
+</Message>
+
+2. Create a stanza for defining the cluster:
+    ```
+    sudo -u postgres pgbackrest --stanza=db-primary stanza-create
+    ```
+
+## Setting up PostgreSQL
+
+1. Change PostgreSQL configurations:
+    ```
+    archive_mode = on
+    archive_command = 'pgbackrest --stanza=db-primary archive-push %p'
+    ```
+
+<Message type="note">
```

Comment:
You need to indent all the code for this message box and the one below, to avoid breaking the chain of numbered steps :-)

Reply:
Oh I saw that now. :( 

I believe I fixed it in b0316ece8f8d856c8ac51c793eea9cad0cbd8b27